### DISCUSSION TITLE
Joining an excalidraw.com collaboration room from the Obsidian plugin

### DISCUSSION THREAD

**Discussion by zsviczian** [excalidraw/excalidraw#4969] [Q&A] [answered]
Hi @ad1992, before investing many hours to understand how the excalidraw-app and CollabWrapper works, I was wondering if it is technically possible to join an https://excalidraw.com collaboration session from within the Obsidian plugin (or any other similar stand-alone implementation). I have two scenarios in mind:
1) a user initiates the collaboration room on excalidraw.com, and some collaborators join from within the Obsidian-Excalidraw local app.
2) the user is able to initiate an excalidraw.com collaboration room from within Obsidian.
Thanks!

**Comment by zsviczian** [comment, marked as answer]
So I answered the question for myself. Both use cases are possible.

All I had to do was to hardcode the production collab server url here:

https://github.com/excalidraw/excalidraw/blob/127af9db23ad9297add8d8fbcf6767f927351d6a/src/excalidraw-app/data/index.ts#L40-L49

https://user-images.githubusercontent.com/14358394/160227349-2222b80b-bf64-4043-9385-798989d1d581.mp4

**Reply by laurensdc** [reply]
Is it possible without being in dev mode?
